tests/pyccel/scripts/kernel/device_test.py

```diff
@@ -1,6 +1,6 @@
 # pylint: disable=missing-function-docstring, missing-module-docstring
 from pyccel.decorators import device, kernel
-from pyccel import cuda
+from pyccel            import cuda
 
 @device
```

Comment:
Does `@device` also need access to the block_idx and thread_idx ?

Reply:
No, I import CUDA for cuda.synchronize()

Reply:
Let me rephrase.

Should it be possible to access the block_idx and thread_idx from a device function? With the current implementation it is only possible to use these functions from a kernel function

Reply:
Yes, and that is another thing to take into consideration

Reply:
I'm going to open an issue about that

Reply:
ok. Please let me know when this is done

Reply:
https://github.com/pyccel/pyccel-cuda/issues/70

Reply:
#70